.github/workflows/ci.yaml

```diff
@@ -155,37 +155,6 @@ jobs:
         with:
           files: lcov.info
 
```

Comment:
x509-cert is `no_std` but it uses alloc, https://github.com/RustCrypto/formats/blob/master/x509-cert/src/lib.rs#L18

I don't recall the full impetus for wanting no alloc in the verify crate.  This may get back to the mention of having a trait for getting access to the certs and keep an alloc crate behind a feature flag

Reply:
The intent was to allow us to communicate between a hardware wallet and a secure enclave. but one thing is that cert verification can just live behind a provider, and we only provide one when "alloc" is enabled.

Reply:
Updated to have an `alloc` feature